Design/SANS/NewISISDesign/Design_Document.md

```diff
@@ -0,0 +1,350 @@
+# Design Document for ISIS SANS Backend Rewrite
+
+This document defines the design  for a new ISIS SANS back-end as required by
+MantidProject issue [#15875](https://github.com/mantidproject/mantid/issues/15875).
+
+The intention of the issue is to replace the organically grown and now difficult to
+maintain back-end of the ISIS SANS reduction work-flow.
+
+### Definition of terms
+
+| Term  | Definition|
+|------|------------|
+|ADS   | Analysis Data Service|
+| PI   | Python Interface |
+
+
+## Motivation
+
+The current SANS reduction system makes use of the so-called `ReductionSingleton`,
+which essentially stores the configurational state either directly, in `ReductionStep`s
```

Comment:
The motivation doesn't include statements regarding the needs of other facilities, and I definitely think that this does need to be broached because Mantid cannot host separate workflows and GUIs for the *ILL*, *ANSTO* and the *ESS* in addition to *ISIS* and the *SNS*. I know that you are considering this, but I'd like to see that flexibility presented as part of the motivation for conducting this work.